Autoupdate potfiles

jkonecny12 opened the pull request:
Use GitHub actions to automatically update pot files in the localization repository. It will daily update the pot file.
After test that this is working as expected I can add more jobs to cover other branches not just master.

I've created new GH action to get our dependencies and run makefile. You can find it here.

 https://github.com/rhinstaller/anaconda-make-executor-action

Badge can't be tested until first run of the action is done. If not working I'll fix it later.

Blocked by: https://github.com/rhinstaller/anaconda/pull/2830

jkonecny12 committed:
Update pot-files automatically from the code repository

Add github action which will daily update pot file.

jkonecny12 committed:
Add badge to README file for automatic pot creation

poncovka reviewed (approved):
Looks good to me. Thanks!

jkonecny12 committed:
Make it possible to manually trigger pot-file-update action

This will be used for development but also if we want to sync pot-file just now
for any reason.

rvykydal reviewed (approved):
Looks good to me
